sugar-event-calendar/includes/admin/settings.php

```diff
@@ -482,3 +499,67 @@ function display_subsection() {
 
 <?php
 }
+
+/**
+ * Check to see if we should be displaying a sidebar.
+ *
+ * @since 2.0.10
+ */
+function maybe_display_sidebar() {
+
+	// Set the date/time range based on UTC
+	$start = strtotime( '2019-11-29 06:00:00' );
+	$end   = strtotime( '2019-12-07 05:59:59' );
+	$now   = time();
+
+	// Only display sidebar if the page is loaded within the date range
+	if ( $now > $start && $now < $end ) {
+		return true;
+	}
+
+	return false;
+}
+
+/**
+ * Output the admin settings sidebar
+ *
+ * @since 2.0.10
+ */
+function display_sidebar() {
+	$coupon_code = 'BFCM2019';
+	?>
+
+	<div class="sc-settings-sidebar">
+
+		<div class="sc-settings-sidebar-content">
+
+			<div class="sc-sidebar-header-section">
+				<img class="bcfm-header" src="<?php echo SC_PLUGIN_URL . 'includes/admin/assets/images/bfcm-header.svg'; ?>">
```

Comment:
Recommend to wrap the `src` in `esc_url()`:

`echo esc_url( SC_PLUGIN_URL . 'includes/admin/assets/images/bfcm-header.svg' );`